test/test_nn.py

```diff
@@ -6513,6 +6513,178 @@ def test_triplet_margin_loss_swap_no_reduce(self):
         self.assertEqual(F.triplet_margin_loss(input1, input2, input3, swap=True, reduction='none'),
                          loss_reference_fns['TripletMarginLoss'](input1, input2, input3, swap=True, reduction='none'))
 
+    def test_triplet_margin_loss_with_distance_parity(self):
```

Comment:
A comment explaining the test would be nice to help future maintainers. This test can be moved to TestNNDeviceType, too, where it will take arguments `(self, device)` and be tested on both CPU and CUDA.

Reply:
(same comment for all the following tests, too)